packages/app-page-builder/src/admin/views/BlockCategories/validators.ts

```diff
@@ -1,7 +1,7 @@
 export const blockCategorySlugValidator = (value: string): boolean => {
-    if (!value.match(/^[a-z]+(\-[a-z]+)*$/)) {
+    if (!value.match(/^[a-z0-9]+(-[a-z0-9]+)*$/)) {
         throw new Error(
-            "Block Category slug must consist of only 'a-z' and '-' characters (for example: 'block-category-slug')"
+            "Block Category slug must consist of only 'a-z', '0-9' and '-' (for example: 'some-slug' or 'some-slug-2')"
```

Comment:
Thanks for addressing this, super annoying. Funny enough, I just reported this a couple of seconds ago in our internal Slack. 🤣 

BTW:

```suggestion
            "Block category slug must consist of only 'a-z', '0-9' and '-' (for example: 'some-slug' or 'some-slug-2')"
```